packages/lesswrong/lib/collections/localgroups/schema.ts

```diff
@@ -85,6 +97,23 @@ const schema: SchemaType<DbLocalgroup> = {
     optional: true,
   },
   
+  categories: {
```

Comment:
this field should have a "hidden: isLWForum", but I think it's better to ship this PR and have me fix that in a subsequent PR

Reply:
Is this resolved?